apps/stable_diffusion/web/utils/global_obj.py

```diff
@@ -0,0 +1,41 @@
+import gc
+import torch
+
+
+def init():
+    global sd_obj
+    global config_obj
+    sd_obj = None
+    config_obj = None
+
+
+def set_sd_obj(value):
+    global sd_obj
+    sd_obj = value
+
+
+def set_cfg_obj(value):
+    global config_obj
+    config_obj = value
+
+
+def set_schedulers(value):
+    global sd_obj
+    sd_obj.scheduler = value
+
+
+def get_sd_obj():
+    return sd_obj
+
+
+def get_cfg_obj():
+    return config_obj
+
+
+def clear_cache():
+    global sd_obj
+    global config_obj
+    del sd_obj
+    del config_obj
+    gc.collect()
+    torch.cuda.empty_cache()
```

Comment:
Do we need torch.cuda ? What happens when there is no cuda

Reply:
I had attempts with separate sd pipeline objects and I remembered I need this probably because I'm with nvidia gpu, but we shouldn't need it now since I just maintain one global pipeline object. Tested and removed.